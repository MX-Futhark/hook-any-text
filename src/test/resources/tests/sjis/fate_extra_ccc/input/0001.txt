8B438E9D82BF82E682AD90B082EA82BD92A982CC92CA8A779848814200FFFFFF00000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D000004000000D814C90800007041000084420000803F0000803F0000000000002000220000002800000080000000FFFFFFFF0002000000010000FFFFFFFF0F000000180000005014C9080814C9087485A348000000000000000082E082A482B882A282D482F192CA82A28AB582EA82BD82CD82B882C882CC82C981410A82A282DC82BE82C98A6F82A682E782EA82C882A292CA8A77984882F095E082AD814200A34800000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F000000200000002100000B0000000000000000000000000000000000000000001F01000000000000000000000000FFFFFFFF008089430000A642000000000000604100000000FFFFFFFF008090430000A642000000000000000000006041FFFFFFFF008089430000C242000000000000604100006041FFFFFFFF008090430000C24200000000FC53A348020000007815C908000000000000000000000000FFFFFFFF000000003C92A348020000009815C908000000000000000000000000FFFFFFFF00010000A492A34802000000B815C908000000000000000000000000FFFFFFFF000100000C93A34802000000D815C908000000000000000000000000FFFFFFFF000100007493A3480200000000000000000000000000000000000000FFFFFFFF00010000DC93A348020000001816C90800000000000000000000000003000100000000004494A348020000003816C9080000000000000000