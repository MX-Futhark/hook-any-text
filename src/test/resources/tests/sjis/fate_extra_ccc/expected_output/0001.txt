["気持ちよく晴れた朝の通学路。",
"もうずいぶん通い慣れたはずなのに、
いまだに覚えられない通学路を歩く。"]